lib/divider/divider.test.tsx

```diff
@@ -48,6 +48,11 @@ describe('<Divider />', () => {
 
     wrapper.setProps({ style: { zIndex: '1' } });
```

Comment:
same here

Reply:
I'm testing 0, 1 and 2 CSS properties across all the assertions—just to be thorough.